webcrypto/test_setup.go

```diff
@@ -9,34 +9,15 @@ import (
 
 	"github.com/dop251/goja"
 	"github.com/stretchr/testify/require"
-	"go.k6.io/k6/js/common"
-	"go.k6.io/k6/js/eventloop"
 	"go.k6.io/k6/js/modulestest"
-	"go.k6.io/k6/lib"
-	"go.k6.io/k6/lib/testutils/httpmultibin"
-	"go.k6.io/k6/metrics"
-	"gopkg.in/guregu/null.v3"
```

Comment:
Dropping this dependancies is breaking the ci - running `go mod tidy` will fix it.